portfolio/src/main/webapp/index2.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Vincent Stevenson</title>
+        <link rel="stylesheet" href="style2.css">
```

Comment:
likewise, style2.css should be renamed to match the new name of this HTML file (so maybe gaming.css)